src/main/java/duke/command/Duke.java

```diff
@@ -0,0 +1,190 @@
+package duke.command;
+
+import duke.task.Deadline;
+import duke.task.Event;
+import duke.task.Task;
+import duke.task.ToDo;
+
+import java.util.Scanner;
+
+public class Duke {
+
+    private static Task[] commandStorage = new Task[100];
+
+    private static final int lengthOfTypeDeadline = 8;
```

Comment:
You can consider adding a Javadoc comment to explain the magic number here `8`. Alternatively, you may wish to have an `enum` class to handle this.